Translates BabyKarte into French

NicolasGrosjean opened the pull request:
**Issue #94**
- Translates JSON data file
- Translates privacy policy
- Remove erased texts in English and German JSON data file

NicolasGrosjean committed:
Remove duplicate localisation

Some keys are duplicates, the first one are replaced by the last one.

So I have deleted the first one to be clearer

NicolasGrosjean committed:
Translates texts into French

Translation done from the English version

NicolasGrosjean committed:
Translates privacy policy into French #94

Discostu36 commented:
Thank you very much for your contribution. I'll have a look at it later today.

ValorNaram reviewed (approved):
Looks good at the technical site. **Well done**

I approve this pull request but we have to wait for @Discostu36's approval too.

After merging it is not directly active and viewable at http://babykarte.github.io Your contribution will be published with the release of v3.0 which is going to happen after #84. I also need to register your contribution to translation files in our system and create the links. I will do that after merge.

Discostu36 commented on privacypolicy-fr.md:
```diff
@@ -0,0 +1,101 @@
+
+La liste suivante indique quels tiers sont utilisés, à quoi ils servent et où vous pouvez trouver leur politique de confidentialité.
+
+## Thiers utilisés
```
```suggestion
## Tiers utilisés
```

NicolasGrosjean replied:
I have forgotten to do a spell checking on this file, I will fix this issue and other one with a spell checker

NicolasGrosjean replied:
Done in 7a4bf3a

Discostu36 reviewed (changes requested):
Just some tiny suggestions.

NicolasGrosjean committed:
Fix spell checking

NicolasGrosjean commented on js/fr.js:
```diff
@@ -0,0 +1,147 @@
+	"PDV_BABYFEEDING_MALE_NO": "Pas de lieu pour les hommes pour nourrir un bébé disponible",
+	"PDV_BABYFEEDING_FEMALE_NO": "Pas de lieu pour les femmes pour nourrir un bébé disponible",
+	"PDV_LEISURE_PLAYGROUND": "Équipements d'aires de jeux",
+	"PDV_PLAYGROUND_SLIDE_YES": "Toboggan",
```
Thanks a lot for the wiki link, I will fix it.

Yesterday I have searched it and I have only found [this page](https://wiki.openstreetmap.org/wiki/FR:Tag:leisure%3Dplayground).

NicolasGrosjean replied:
Done in 3ae64d7

NicolasGrosjean committed:
Fix French translation

Replace some element by the wiki one : https://wiki.openstreetmap.org/wiki/FR:Key:playground

NicolasGrosjean commented on js/fr.js:
```diff
@@ -0,0 +1,147 @@
+	"PDV_HOUSENUMBER_UNKNOWN": "?",
+	"PDV_ZIPCODE_UNKNOWN": "Code postal inconnu",
+	"PDV_TITLE_HOME": "General",
+	"PDV_TITLE_BABY": "Baby friendliness",
```
Yes I have skipped this line because it is difficult to translate.

There is no translation in French for the word _friendly_ so according of the context we do a periphrasis (for example _Adapté aux bébés_ which is in fact the translation of _Suitable for babies_) or we let the word _friendly_ (for example _Bébé friendly_).

I will search the context to chose the best option

NicolasGrosjean replied:
Context : 
![image](https://user-images.githubusercontent.com/13560473/60284222-4ef6d500-990b-11e9-8bf9-a5d39b61790e.png)

The menu show in fact the existence or not of equipment. So I will translate _Baby friendliness_ by _Équipements pour bébés_ .

NicolasGrosjean replied:
Done in 66321d7

Discostu36 replied:
The stroller key is not about equipment. So I think Adapté aux bébés might actually be the better translation.

NicolasGrosjean replied:
You're right, I have fixed it in af8e38c

NicolasGrosjean committed:
Fix missing French translation

Translate a difficult expression to translate

NicolasGrosjean committed:
Fix friendliness translation

Friendliness is difficult to translate into French.
I have tried to use context to translate it (equipment) but I am wrong about this context.
Stroller friendliness is not an equipment.

Discostu36 commented:
Thank you very much! Are you interested in joining our team? There will certainly be new strings to translate in the future or maybe there are other things you are interested to work at. No pressure, just asking 🙂

NicolasGrosjean commented:
Of course I can translate new strings in the future.
For other things, why not, but my skills in front-end are very limited.

Binnette commented:
Thank you @NicolasGrosjean for translating BabyKarte to french 👍
